### PR TITLE
docs(aio): remove _blank target

### PR DESCRIPTION
**Pull request by Foxandxss**
@IgorMinar requested to not use `target="_blank"` since that doesn't work well on mobile.

### CODE REVIEW

File: aio/content/guide/browser-support.md [hunk 1]
```diff
@@ -371,7 +371,7 @@ Here are the features which may require additional polyfills:
   <tr style="vertical-align: top">
  
     <td>
-       <a href="../api/common/index/NgClass-directive.html" target="_blank">NgClass</a>      <span> on SVG elements</span>
+       <a href="../api/common/index/NgClass-directive.html">NgClass</a>      <span> on SVG elements</span>
```
Comment (Foxandxss) [inline aio/content/guide/browser-support.md]: this one is the wrong link, I will fix that in a separate PR.

**Review by gkalpak — APPROVED** [review]
LGTM (as long as you fix that link in another PR 😛)

**Comment by wardbell** [comment]
Failed travis task is irrelevant flake. Let's merge.

**Comment by mhevery** [comment]
@Foxandxss can't merge due to conflicts. Please rebase.

**Pull request merged**